.github/mergify.yml

```diff
@@ -12,8 +12,8 @@ pull_request_rules:
     conditions:
       - and:
         - "-draft"
-        - "#check-failure=0"
```

Comment:
this is not needed anymore?

Reply:
Nope, because of our branch protection rules